packages/tag/src/components/Tag.story.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withInfo } from "@storybook/addon-info";
+import { action } from "@storybook/addon-actions";
+
+import Tag from "./Tag";
+
+storiesOf("Removable-tags", "module")
+  .add(
+    "Default",
+    withInfo()(() => (
+      <div style={{ marginTop: "20px" }}>
+        <Tag onRemove={action("onRemove")} text="THIS IS A TAG" />
+      </div>
+    ))
+  )
+  .add(
+    "Not removable",
```

Comment:
I think default tab is not removable... just pass a text and you have a tag.
So...
- [ ] rename this story to `Removable`
- [ ] remove `onRemove` property from `Default` story